packages/amplify-storage-simulator/src/__tests__/S3server.test.ts

```diff
@@ -169,10 +176,40 @@ describe("Test put api", () => {
   test(" multipart upload", async () => {
     const params = {
       Bucket: bucket, // pass your bucket name
-      Key: "upload/long_image.jpg", 
+      Key: "upload/long_image.jpg",
       Body: buf2
     };
     const data = await s3client.upload(params).promise();
     expect(data.Key).toBe("upload/long_image.jpg");
   });
+
+  test(" multipart upload", async () => {
+    const params = {
+      Bucket: bucket, // pass your bucket name
+      Key: "upload/long_image1.jpg",
+      Body: buf2
+    };
+    const data = await s3client.upload(params).promise();
+    expect(data.Key).toBe("upload/long_image1.jpg");
+  });
+
+  test(" multipart upload", async () => {
```

Comment:
Why 3 test repeat the same thing? Are you  trying concurrent uploads? These tests run in series and it wouldn't be a concurrent upload if thats the case

Reply:
made this as a one test with name async uploads